dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Rules/MagicNumber/MagicNumberAnalyzer.cs

```diff
@@ -12,9 +14,10 @@ public sealed class MagicNumberAnalyzer : DiagnosticAnalyzer
     {
         public const string Id = DiagnosticId.MagicNumber;
 
-        private const string Title = "Variable declaration uses a magic number";
+        private const string Title = "Variable declaration uses a magic number"; 
```

Comment:
are whitespace changes here intentional? Just thinking we might get additional 
build warnings (I _think_ we have warnings as errors, but can't remember)

Reply:
not intentional will remove